data/human/human missions.txt

```diff
@@ -6890,3 +6890,1716 @@ mission "Hogshead's Dilemma II"
 			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
 			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
 				decline
+
+
+
+mission "Timothy Radrickson 1: Stowaway"
+	invisible
+	landing
+	source
+		near "Zeta Aquilae" 3 10
+		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
```

Comment:
Some quotes could be removed here.

Reply:
This might be something that has changed in the two years between when I wrote this part of the mission, and now. But I don't actually know what quotes need be removed here, or how it should look without them.

Reply:
The quotes aren't necessary, but for gov'ts they seem to be all in quotation marks in the data files.